package.json

```diff
@@ -31,7 +31,7 @@
     "typecheck": "tsc",
     "prepublishOnly": "nr build",
     "release": "bumpp && pnpm publish --no-git-checks",
-    "test": "vitest"
+    "test": "unbuild && vitest"
```

Comment:
The new test case are test against the `dist`, so it needs to be built before the test happens.